src/contracts/permissions/Pausable.sol

```diff
@@ -32,35 +32,32 @@ contract Pausable is IPausable {
 
     /// @notice
     modifier onlyPauser() {
-        require(pauserRegistry.isPauser(msg.sender), "msg.sender is not permissioned as pauser");
+        require(pauserRegistry.isPauser(msg.sender), OnlyPauser());
         _;
     }
 
     modifier onlyUnpauser() {
-        require(msg.sender == pauserRegistry.unpauser(), "msg.sender is not permissioned as unpauser");
+        require(msg.sender == pauserRegistry.unpauser(), OnlyUnpauser());
         _;
     }
 
     /// @notice Throws if the contract is paused, i.e. if any of the bits in `_paused` is flipped to 1.
     modifier whenNotPaused() {
-        require(_paused == 0, "Pausable: contract is paused");
+        require(_paused == 0, CurrentlyPaused());
         _;
     }
 
     /// @notice Throws if the `indexed`th bit of `_paused` is 1, i.e. if the `index`th pause switch is flipped.
     modifier onlyWhenNotPaused(
         uint8 index
     ) {
-        require(!paused(index), "Pausable: index is paused");
+        require(!paused(index), CurrentlyUnpaused());
```

Comment:
Think this should be `CurrentlyPaused()` since that is the behaviour breaking it.

Reply:
oops will fix